### PR TITLE
Add Note About Designing for Device Independence

### LINKED ISSUE #126: Add Note About Device Independence
We need a note about designing for device-independence:
https://github.com/ucsb-wsg/web-accessibility/blob/master/web-accessibility-guidelines.md#7-design-for-device-independence

This could be added to the current "Web Accessibility" section.

### PR DESCRIPTION
**Pull request by loganfranken**
Fixes #126

### CODE REVIEW

**Comment by tenken** [comment]
:+1:

**Comment by loganfranken** [comment]
Hey there! Anything I can do to help merge this?

Let me know; thanks!

**Comment by loganfranken** [comment]
Good morning! Any updates on this one?

File: web-accessibility.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ permalink: /web-accessibility/
...
 
+### General Web Accessibility Guidelines
+
+In addition to the PDF above and the recommendations described throughout the Web
+Guide, the Web Standards Group also provides the following general guidelines for
+addressing accessibility in your websites:
+
+**Your page should not depend on the user using a specific type of device**.
+In other words, your page should be navigable by mouse, keyboard, voice, and
+head wand. Elements that appear only upon mouse-over, for example, are usually
```
Comment (garster) [inline web-accessibility.md:17]: Can I suggest adding "touch" to the list. I hate when I cant use mouse-over dropdowns on a tablet.

**Comment by loganfranken** [comment]
Updated! Thanks @garster!

**Pull request merged**